src/schema/rules/sidecars/pet.yaml

```
@@ -176,6 +180,25 @@ PETReconstruction:
     SinglesRate: recommended
     RandomRate: recommended
 
+# set required reconstruction fields to optional when "none" is provided as a parameter label
+EntitiesReconMethodMetadata:
+  selectors:
+    - modality == "pet"
+    - suffix == "pet"
+    - count(sidecar.ReconMethodParameterLabels, "none") >= 1
```

```suggestion
    - "none" in sidecar.ReconMethodParameterLabels
```

I _think_ this is the preferred way of saying "X contains 'string'"

---

I had originally done it that way, but changed it after reading over Chris' updated schema readme. It wasn't clear to me whether or not a an element in an array was a subfield or not. I felt that using count made it much more clear that "none" was an element in an array. Perhaps I just need to update the documentation clarify that the `in` operator works this way on arrays.